pkg/crc/machine/machine.go

```diff
@@ -40,6 +41,8 @@ import (
 	"github.com/code-ready/machine/libmachine/state"
 )
 
+const MinimumMemoryForMonitoring = 13671
```

Comment:
How did we get this value?

Reply:
It comes from a misunderstading. Doc says 14336, I will change it.

However, there are mistakes in the code: we use `units.HumanSize` (base 1000) instead of `units.BytesSize` (base 1024). I will add a commit to fix this.

```
// HumanSize returns a human-readable approximation of a size
// capped at 4 valid numbers (eg. "2.746 MB", "796 KB").
func HumanSize(size float64) string {
	return HumanSizeWithPrecision(size, 4)
}

// BytesSize returns a human-readable size in bytes, kibibytes,
// mebibytes, gibibytes, or tebibytes (eg. "44kiB", "17MiB").
func BytesSize(size float64) string {
	return CustomSize("%.4g%s", size, 1024.0, binaryAbbrs)
}
```

Reply:
It turns out to be more complicated for df. I will do that in a separated PR.